bootstrap/bungeecord/src/main/java/org/geysermc/geyser/platform/bungeecord/GeyserBungeePingPassthrough.java

```diff
@@ -59,7 +61,17 @@ public GeyserPingInfo getPingInformation(InetSocketAddress inetSocketAddress) {
                 future.complete(event);
             }
         }));
-        ProxyPingEvent event = future.join();
+
+        ProxyPingEvent event;
+
+        try {
+            event = future.get(100, TimeUnit.MILLISECONDS);
+        } catch (Throwable cause) {
+            GeyserImpl.getInstance().getLogger().error("Failed to get ping information for " + inetSocketAddress, cause);
+
```

Comment:
would probably be wise to do something similar to this: https://github.com/GeyserMC/Geyser/blob/9cdda70/core/src/main/java/org/geysermc/geyser/session/GeyserSession.java#L1092